internal/rpc/handlers.go

```diff
@@ -6,9 +6,9 @@ import (
 	"net/http"
 
 	"github.com/labstack/echo/v4"
-	"github.com/pokt-network/pocket/app"
-	"github.com/pokt-network/pocket/shared/codec"
-	typesUtil "github.com/pokt-network/pocket/utility/types"
+	app "github.com/pokt-network/pocket/cmd"
```

Comment:
This seems a bit weird.

If `app` is a good semantic name for it, then I'd argue we shouldn't rename `cmd` to `app`.

Why the change?